CMakeLists.txt

```diff
@@ -36,6 +36,8 @@ option(TRITON_ENABLE_STATS "Include statistics collections in backend" ON)
 set(TRITON_TENSORFLOW_VERSION "1" CACHE STRING "TensorFlow version, must be '1' or '2'")
 set(TRITON_TENSORFLOW_DOCKER_IMAGE "nvcr.io/nvidia/tensorflow:20.08-tf1-py3" CACHE STRING
     "Docker image containing the TensorFlow build required by backend")
+set(TRITON_TENSORFLOW_LIB_PATHS "" CACHE PATH "Paths to TensorFlow libraries")	
```

Comment:
Add "Multiple paths may be separated with a semicolon." to description of both of these new vars.

Reply:
Done